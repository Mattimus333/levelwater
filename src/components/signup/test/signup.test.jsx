import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Signup from '../signup.jsx';


describe('signup', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<Signup />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
