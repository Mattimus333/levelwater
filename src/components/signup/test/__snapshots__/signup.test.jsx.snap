exports[`signup renders a snapshot 1`] = `
<div
  id="signup">
  <Connect(ReduxForm) />
  <OAuthButton />
</div>
`;
